src/python/pants/core/goals/fmt.py

```diff
@@ -108,14 +108,15 @@ def message(self) -> str | None:
         message = "made changes." if self.did_change else "made no changes."
 
         # NB: Instead of printing out `stdout` and `stderr`, we just print a list of files which
-        # changed. We do this for two reasons:
+        # were changed or added. We do this for two reasons:
         #   1. This is run as part of both `fmt` and `lint`, and we want consistent output between both
         #   2. Different formatters have different stdout/stderr. This way is consistent across all
         #       formatters.
```

Comment:
```suggestion
        #       formatters.
        # We also allow added/removed files because the `fmt` goal is more like a `fix` goal.
        # (See https://github.com/pantsbuild/pants/issues/13504)
```